packages/ide-extension/src/enhancement/enhancements.json

```diff
@@ -1,4 +1,16 @@
 {
     "nodeEnhancements": [],
-    "htmlEnhancements": []
+    "htmlEnhancements": [
+        {
+            "text": "Fiori: Archive Project",
+            "command": {
+                "label": "Archive Project",
+                "description": "Run command to archive a project from current workspace",
```

Comment:
How about `Run command to archive a Fiori tools project from current workspace`

Reply:
fixed